### DISCUSSION TITLE
Adjusting homerow order

### DISCUSSION THREAD

**Discussion by matchai** [manna-harbour/miryoku#34] [Q&A] [answered]
Hi there,
In playing around with miryoku and am loving the pre-built configuration. The only thing I'd like to tweak, which I'm sure is a common need, is the homerow order, in order to better fit with the modifier usage on macOS.

I think the ideal way to support this would be by allowing people to homerow order labels used here:
https://precondition.github.io/home-row-mods#home-row-mods-order

```
// Set homerow order to ⎈⎇◆⇧ (CAGS)
MIRYOKU_HOMEROW_ORDER = CAGS
```

**Comment by manna-harbour** [comment, marked as answer]
Thanks for the suggestion!  There's a built-in feature in QMK for swapping Ctrl / Gui for Mac, but not in ZMK.  You might like to create a ZMK issue for it as it would be a useful feature in general.

I've added it to the list of mods customisations to be supported in future, see #33.

**Reply by mybuddymichael** [reply]
Thanks for the tip about that QMK feature! I didn't know about that. But I do still think custom ordering could be useful as I think the ideal order on Mac is ACGS based on usage. Thanks so much for your work on this!

**Reply by mybuddymichael** [reply]
In the meantime I just edited the keymap directly to get the order I want. (I am very new to the custom keyboard scene and QMK, and I didn't immediately realize how easy this is until today. 😬 )

**Reply by manna-harbour** [reply]
Great, I was just about to mention that! 😅  There's a plan to make it easy to substitute layers, but the mods are on all layers, so you'd have to replace everything. It would be a [one line](https://github.com/manna-harbour/miryoku_babel/blob/7eb2ea0b5e361e43c2ced1b2273f89fce284b98f/readme.org?plain=1#L110) change to only the hold table in Miryoku Babel though.

**Reply by mybuddymichael** [reply]
Ah, I was wondering why my other layers were acting funky! I see now.
